extension/src/popup/components/accountMigration/ReviewMigration/index.tsx

```diff
@@ -144,67 +133,83 @@ const AccountListItems = ({
   return accountListItems.length ? (
     <>
       {accountListItems.map((acct) => (
-        <section className="ReviewMigration__section" key={acct.publicKey}>
-          {acct.isReadyToMigrate ? (
+        <MigrationReviewListSection
+          isUnfunded={!acct.xlmBalance}
+          key={acct.publicKey}
+        >
+          {acct.xlmBalance ? (
             <>
-              <div className="ReviewMigration__row ReviewMigration__account-row">
```

Comment:
most of the work in this file is breaking out sections into resuable components as this screen shares a lot of similarities with the `migration-complete` screen